packages/tasks/src/tasks/mask-generation/about.md

```diff
@@ -0,0 +1,43 @@
+## Use Cases
+
+### Filtering an Image
+
+The mask might serve as an initial filter to eliminate irrelevant information. For instance, when monitoring vegetation in satellite imaging, a mask can be created to identify green spots, highlighting the relevant region of the image.
+
+### Pre-processing
+
+Generating masks that can to separate background and foreground (or other interesting properties) in an image can be done to facilitate learning, especially in semi- or unsupervised learning.
+
+### Human-in-the-loop
+
+For applications where humans are in the loop, masks highlight certain region of images for humans to validate. 
+
+## Task Variants
+
+### Binary Masks
+
+In binary masks, one region is labeled with ones (1) while another region is labeled with zeroes (0). 
+In this case, matrix multiplication easily retrieves the region of interest.
+
+### Non-Binary Masks
+
+In non-binary masks, the mask contains more than two classes. This is useful when more than one mask is generated for the same image since it makes dealing with the image easier. If there are non-overlapping labeled regions, it is straightforward to recover the mask by selecting which pixels in the image have the corresponding region. If they overlap, a unique way to recover a region might have to be put in place.
+
+### Segmentation Mask
```

Comment:
I think this is not a use-case but due to how mask generation is sort of another variant of segmentation and segmentation would be the bigger part IMO